main/wallet-backend.js

```diff
@@ -33,6 +34,14 @@ const BERYX_TOKEN = 'eyJhbGciOiJFUzI1NiIsImtpZCI6ImtleS1iZXJ5eC0wMDEiLCJ0eXAiOiJ
 // eslint-disable-next-line @typescript-eslint/no-empty-function
 async function noop () {}
 
+/**
+ * @param {string} value
+ * @returns {value is `f1${string}`}
+ */
+function isF1Address (value) {
+  return value.startsWith('f1')
```

Comment:
this check is giving false confidence, I would expect it to also try to parse the address

Reply:
I see what you mean. How about renaming the function to `startsWithF1`?

FWIW, here is the implementation of the other type guard function were are using:

```js
export function isEthAddress(address) {
  return ethers.isAddress(address) && Number(address) !== 0;
}
```

> hank you for taking care of this update! I shied away from this when I saw the typescript errors.

Please bring such PRs to my attention. I am ignoring most Dependabot PRs.

Reply:
or what about `return ethers.isAddress(address) && address.startsWith('f1')`?

Reply:
I guess all glif cares about from a type perspective is that it starts with f1, so `startsWithF1` also makes sense to me

Reply:
Renamed in [098d5d2](https://github.com/filecoin-station/desktop/pull/1527/commits/098d5d2e745b18977b61b6cb8afea9436496d755). LGTY now?